### DISCUSSION TITLE
Running a php script with ray

### DISCUSSION THREAD

**Discussion by K2ouMais** [spatie/ray#308] [Q&A]
Hello everyone,

I am having problems with debugging a simple php script/class.

```php
namespace App\Support;

class MyTestClass
{
    public function testMethod(string $testVar): string
    {
        ray($testVar)->red();
        return $testVar;
    }
}

echo (new MyTestClass())->testMethod('This is a Test');
```

Now if I run this with Ctrl+Shift+F10 on PHPStorm or if I try to run it on the Console I get the following error.

```PHP Fatal error:  Uncaught Error: Call to undefined function App\Support\ray()```

Am I missing something?

Thanks in advance

**Comment by patinthehat** [comment]
@K2ouMais Are you using the composer autoloader?  It should autoload the helpers file that contains the `ray()` function.
However, you can try using:
```php
\ray($testVar)->red();
````

**Reply by K2ouMais** [reply]
Yes I am using the composer autoloader and I still get the error.

Even when I try ```\ray('hello');``` it doesn't work.

**Reply by patinthehat** [reply]
@K2ouMais Instead of using the helper function, does creating the class directly work?
```php
(\Spatie\Ray\Ray::create())->send('hello');
```

**Reply by K2ouMais** [reply]
No it still doesnt work...

![Screenshot 2021-02-14 092715](https://user-images.githubusercontent.com/52510455/107871984-ecb76300-6ea6-11eb-8804-ac6a9a9c9760.png)

**Reply by freekmurze** [reply]
Please provide us with a minimal app to demonstrate the problem

**Reply by K2ouMais** [reply]
Hello Freek,

just install a fresh Laravel project and put a Support folder inside App.

Install the ray package through composer and that is all you need.

Try my class already posted here and you will see the error when running it through PHPStorm or the CLI.

There isnt any more into that.

**Reply by freekmurze** [reply]
For me that works. Please submit a link to a minimal app that does for work for you.
